CHANGELOG

```diff
@@ -0,0 +1,53 @@
+2023-12-28 16:06 - v1.0.23 - Customized the Username
```

Comment:
How would I go about removing a specific commit?

Reply:
It is possible but more advanced.
I don't plan to merge this PR (because it is too big) but instead make smaller PRs out of this big one. When I do that I simply won't add certain commits. So, don't worry about this for now.